src/main/java/com/google/api/codegen/viewmodel/metadata/PackageMetadataView.java

```diff
@@ -88,6 +88,9 @@ public String resourceRoot() {
   @Nullable
   public abstract String versionPath();
```

Comment:
I know we haven't been consistent about this previously, but it would be useful to have javadoc either on the builder or the accessor.